programs/cleos/main.cpp

```diff
@@ -249,6 +249,15 @@ chain::action generate_nonce_action() {
    return chain::action( {}, config::null_account_name, "nonce", fc::raw::pack(fc::time_point::now().time_since_epoch().count()));
 }
 
+void prompt_for_wallet_password(string& pw, const string& name) {
```

Comment:
SecureEnclave wallet ignores the password field for the unlock & remove_key actions because it always uses the local system's authentication (this way we can use fingerprint authentication for it). This cheesy check makes sure we don't prompt for a useless password for this wallet